cluster-provision/images/vm-image-builder/fedora-with-test-tooling/cloud-config

```diff
@@ -43,7 +43,14 @@ write_files:
         bind 'set enable-bracketed-paste off'
 runcmd:
   - sudo systemctl daemon-reload
-  - sudo dnf install -y kernel-modules-$(uname -r) qemu-guest-agent stress-ng dmidecode virt-what vm-dump-metrics tpm2-tools nc sg3_utils iperf3
+  - sudo dnf install -y kernel-modules-$(uname -r) qemu-guest-agent stress-ng dmidecode virt-what vm-dump-metrics tpm2-tools nc sg3_utils iperf3 wget
```

Comment:
Note that `wget` is added.

Reply:
Yes, I add `wget`. Do we have any concerns here?

Reply:
I got it wrong thinking it was added by a mistake, sorry for the noise.
I think will be nicer to keep the image light as possible using existing tools, maybe curl instead.

Reply:
> I think will be nicer to keep the image light as possible using existing tools, maybe curl instead.

Yes, I agree with you. I have two reasons for install `wget`.
1. we have a e2e test that use `wget`, https://github.com/kubevirt/kubevirt/blob/bd22e81c15a422359735cd73c23d7bd0142ae103/tests/network/networkpolicy.go#L391
2. Another reason is that, if the resource is not available, `curl` will not return error but wget will. I use `wget` here in case the link of busybox netcat is not available.

Reply:
Hi, @ormergi , what's your opinion?